src/utils/plotData.js

```diff
@@ -0,0 +1,34 @@
+export default class PlotData {
+    points = [];
+    charts = {};
+
+    constructor(title) {
+        this.title = title;
+    }
+
+    addPoint(timestamp, values) {
+        let diff = Object.keys(values)
+            .filter(v => !Object.keys(this.charts).includes(v));
+        if (diff.length) {
+            throw new Error(diff + " chart names not defined");
```

Comment:
"Magic strings" should not be used - extrac it to constant.

Reply:
@StencelMichal